test/bootstrap.js

```diff
@@ -1,10 +1,15 @@
+import { assert } from 'chai';
 import { configure } from 'enzyme';
 import { Adapter } from 'enzyme-adapter-preact-pure';
 import 'preact/debug';
+import sinon from 'sinon';
 
 // Expose the sinon assertions.
 sinon.assert.expose(assert, { prefix: null });
 
+globalThis.assert = assert;
+globalThis.sinon = sinon;
```

Comment:
Karma sets `sinon` and `assert` as globals via plugins. This runner does not have equivalent plugins, so I set them here for the sake of having a working example.

We can evaluate later how to proceed.